modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,9 +19,21 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {ProductShortName} fixed issues
 
+* link:https://issues.redhat.com/browse/MAISTRA-2010[MAISTRA-2010] AuthorizationPolicy does not support `request.regex.headers` field. The `validatingwebhook` rejects any AuthorizationPolicy with the field, and even if you disable that, Pilot tries to validate it using the same code, and it does not work. The issue was fixed in 1.1, but not 2.0.
```

Comment:
"The issue was fixed in 1.1, but not 2.0" sounds a bit like we didn't fix the issue after all. maybe just mention that this was a regression? it is fixed in both versions now